docs/ref/files/storage.txt

```diff
@@ -95,6 +95,15 @@ The Storage Class
         able to return the last accessed time this will raise
         ``NotImplementedError`` instead.
 
+    .. method:: copy(src_name, dst_name)
+
+        Copy the file specified by ``src_name`` to ``dst_name``. If
+        ``dst_name`` already exists ``ValueError`` will be raised. For
+        storage systems that don't support copy this will raise
+        ``NotImplementedError`` instead.
+
+        .. versionadded:: 1.8
```

Comment:
Well, this means that I also need to move the changes to the release notes in an 1.9.txt

However, that file doesn't exist yet. Is there any template for creating release notes for a new version?